### PR TITLE
Quiz ui fix

### LINKED ISSUE #6: Update Quiz page UI

### PR DESCRIPTION
**Pull request by rpasricha45**
1. added matieral ui usestyle to Questions components , to style and fix issue 
2. added Paper matieral UI component , to allow for scrolling for the quiz page

### CODE REVIEW

File: src/components/Question.js [hunk 1]
```diff
@@ -4,9 +4,31 @@ import RadioGroup from '@material-ui/core/RadioGroup'
...
+	  // label + control group use the entire width of the parent element
+	  display: "flex",
+	  // component default is "flex-start", using "space-between" pushes
+	  // both flexed content to the right and left edges of the flexbox
+	  // Note: the content is aligned to the right by default because
+	  // the 'labelPlacement="start"' component prop changes the flexbox
+	  // direction to "row-reverse"
+	  justifyContent: "space-between",
+	  width:'250px',
+
```
Comment (vkoli) [inline src/components/Question.js:14]: Comments should be removed
Space between answer label and radio button is ok.

File: src/components/Question.js [hunk 2]
```diff
@@ -23,42 +45,49 @@ const Question = (props) => {
 				name={props.questionName}
```
Comment (vkoli) [inline src/components/Question.js:36]: Can you also look into increasing the size the Question text in the Form Label component?
At Line 42.

File: src/components/Quiz.js [hunk 3]
```diff
@@ -72,13 +73,17 @@ const Quiz = (props) => {
...
   return (
     <div className="Quiz">
+      <Paper style={{maxHeight: 850, overflow: 'auto'}}>
+
+
       <ul list-style-type="none">
         {questionList.map((item, id) => (
           <li key={id}>
             <Question questionName={item} update={handleUpdate} />
           </li>
```
Comment (vkoli) [inline src/components/Quiz.js]: Is it possible to get ride of the list-style? Right now its showing bullet points which doesn't really look clean necessarily. 
If not, just change the li tag to ol for numbered list.

**Review by vkoli — CHANGES_REQUESTED** [review]
A few additional points:
1) Add in you change to App.js in this commit
2) Is it possible to have the Quiz div take up the remaining screen width? Right now the component stops somewhere before.

**Comment by rpasricha45** [comment]
I removed the comments , + bullpoints , and made the overall quiz wider to make look nicer ,  I  didn't get to make the text bigger yet stuck on this part

**Comment by vkoli** [comment]
What we have for the quiz ui right now is sufficient for this week, so we can close this branch

**Pull request merged**